smart-contracts/test/Lock/erc721/Non_Public_transferFrom.js

```diff
@@ -1,37 +1,26 @@
-
-const Units = require('ethereumjs-units')
-const Web3Utils = require('web3-utils')
-
 const deployLocks = require('../../helpers/deployLocks')
-const shouldFail = require('../../helpers/shouldFail')
 const Unlock = artifacts.require('../../Unlock.sol')
 
-let unlock, locks
+let unlock
 
 contract('Lock ERC721', (accounts) => {
-  before(() => {
-    return Unlock.deployed()
-      .then(_unlock => {
-        unlock = _unlock
-        return deployLocks(unlock)
-      })
-      .then(_locks => {
-        locks = _locks
-      })
+  before(async () => {
+    unlock = await Unlock.deployed()
+    await deployLocks(unlock)
   })
 
   // from  transferFrom.js, ln#59:
   it.skip('should abort if the lock is private', async () => {
-    await shouldFail(locks['PRIVATE']
-      .transferFrom(from, to, tokenId, {
-        from
-      }), '')
+    // await shouldFail(locks['PRIVATE']
```

Comment:
since this is skipped, why comment it?

Reply:
b/c it's an error.  `from`, `to` etc do not exist.  the linter notices that.  It seems we should delete it, comment it out, or fix it.

Reply:
Dang. ok  commenting is fine for now I guess :/